lib/modules/manager/argocd/extract.ts

```diff
@@ -18,6 +19,8 @@ import {
 } from './schema';
 import { fileTestRegex } from './util';
 
+const kustomizeImageRe = regEx(/(.+)=(.+):(.+)/);
```

Comment:
this is wrong. it requires a tag and doesn't support digests. tag and digest should be optional 

you should use a similar approach like we already use in kustomize